lib/components/SignInForm.jsx

```diff
@@ -14,24 +15,55 @@ import PasswordField from "./PasswordField";
  * Optional props:
  * - legend: string. sets the form legend
  * - id: string. Sets the form's id attribute
- * - inputClass: string. Adds any custom classes to TextInput fields
  */
-export default function SignInForm ({legend, input}) {
-  return (
-    <Form>
-      <fieldset className="usa-fieldset-inputs usa-sans">
-        <legend className="usa-drop_text">{legend}</legend>
-        <span>or <a href="javascript:void(0);">create an account</a></span>
-        <TextInput
-        label="Username or email address"
-        id="username"
-        required />
-        <PasswordField />
-      </fieldset>
-    </Form>
-  )
+export default class SignInForm extends React.Component {  /**
```

Comment:
@n8mellis should this be broken down in to smaller components, or should it be handled as a part of SignInForm component - button texts, show/hide password toggle?